Source/WebCore/workers/service/server/SWServer.cpp

```diff
@@ -201,7 +201,8 @@ void SWServer::addRegistrationFromStore(ServiceWorkerContextData&& data, Complet
     LOG(ServiceWorker, "Adding registration from store for %s", data.registration.key.loggingString().utf8().data());
 
     auto registrableDomain = WebCore::RegistrableDomain(data.registration.key.topOrigin());
-    validateRegistrationDomain(registrableDomain, ServiceWorkerJobType::Register, m_scopeToRegistrationMap.contains(data.registration.key), [this, weakThis = WeakPtr { *this }, data = WTFMove(data), completionHandler = WTFMove(completionHandler)] (bool isValid) mutable {
+    auto registrationKey = data.registration.key;
```

Comment:
optional nit - you could also use this variable above in `registrableDomain`.

Reply:
Fixed. Thank you.